pal03377/Athena-New
# ThemisML Module

This PR introduces a new ThemisML module based on https://github.com/ls1intum/Themis-ML

Currently, it's IMO not fast enough for production usage, but the suggestion algorithm already works for whole methods.

To test: 
- Send submissions of the "Hello World" exercise
- Send feedback of the "Hello World" exercise
- Request feedback suggestions for the submission with ID 104, which references the same code as submission 101 with feedback (a bit simplistic, I know - but works for basic testing)
- You should get back a suggestion:
<img width="715" alt="screenshot-2023-06-11_001427" src="https://github.com/pal03377/Athena-New/assets/9006596/50efb05b-73cc-4f2e-9d51-e6b6e1960e9f">


I also changed the feedback schemas and models in a way that could already solve #22 in the process (closes #22)
